src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -333,8 +328,9 @@ object ICollection.SyncRoot
         #region Subclass helpers
 
         //virtual function for subclass that needs to update count
-        internal virtual void VersionCheck() { }
-
+        internal virtual void VersionCheck()
+        {
+        }
```

Comment:
FWIW, I'm not a fan of forcing empty brackets to take up their own lines like this. Is this in the style guide?